Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -4534,6 +4534,19 @@ AtomString consumeFamilyNameRaw(CSSParserTokenRange& range)
     return concatenateFamilyName(range);
 }
 
+Vector<AtomString> consumeFamilyNameList(CSSParserTokenRange& range)
```

Comment:
This seems a bit of a surprising addition. Why aren't we sharing code with the existing infrastructure to parse font-family?

Reply:
`font-family` parses both <family-name> and <generic-name>.

Here, we exclusively parse <family-name> : https://drafts.csswg.org/css-fonts-4/#family-name-value

Reply:
I've added a comment on the prototype in the header file.